hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/debug/replicas/chunk/ChunkKeyHandler.java

```diff
@@ -177,12 +186,17 @@ protected void execute(OzoneClient client, OzoneAddress address)
           xceiverClientManager.releaseClientForReadData(xceiverClient, false);
         }
       }
-      result.set("KeyLocations", responseArrayList);
+      result.set("keyLocations", responseArrayList);
       String prettyJson = JsonUtils.toJsonStringWithDefaultPrettyPrinter(result);
       System.out.println(prettyJson);
     }
   }
 
+  private String convertBase64ToHex(String base64Checksum) {
```

Comment:
We should `StringUtils#byteToHexString` directly instead of making a new method.